chia/server/start_service.py

```diff
@@ -79,9 +79,9 @@ def __init__(
         chia_ca_crt, chia_ca_key = chia_ssl_ca_paths(root_path, self.config)
         inbound_rlp = self.config.get("inbound_rate_limit_percent")
         outbound_rlp = self.config.get("outbound_rate_limit_percent")
-        if NodeType == NodeType.WALLET:
+        if node_type == NodeType.WALLET:
```

Comment:
oh limit was not raised previously at all

Reply:
yeah. we are still hitting the limit with 60 sometimes though because the minute barriers can be at different points, so we reset our sent count but the node has not reset theirs yet.

Reply:
thought incoming was at 100% ? or am I missing how it works

Reply:
it's 100% but it resets every minute. The thing is that the point that it resets might not be the same for the client and server

Reply:
and 0.6*2 > 1.0